Makediff for koppel in /home/faculty/koppel/r/gbs/Chemora/repos/carpet
diff --git a/CarpetLib/src/coeffs_prolongate_3d_stagger.hh b/CarpetLib/src/coeffs_prolongate_3d_stagger.hh
index 4325128..eff946e 100644
--- a/CarpetLib/src/coeffs_prolongate_3d_stagger.hh
+++ b/CarpetLib/src/coeffs_prolongate_3d_stagger.hh
@@ -18,7 +18,7 @@
 // Similarly for f(x) = k x^2 and f(x) = k x^3, so we have a cubic polynomial
 // fit
 
-double coeff[10][10];
+T coeff[10][10];
 
 if (ORDER == 2) {
   coeff[1][1] = 5.0 / 32.0;  // i-1
diff --git a/CarpetLib/src/coeffs_restrict_3d_stagger.hh b/CarpetLib/src/coeffs_restrict_3d_stagger.hh
index 793e0aa..04946fd 100644
--- a/CarpetLib/src/coeffs_restrict_3d_stagger.hh
+++ b/CarpetLib/src/coeffs_restrict_3d_stagger.hh
@@ -30,8 +30,8 @@
 
 #define ORDER_STAG 3
 
-double coeff[10];
-double coeff_i[10];
+T coeff[10];
+T coeff_i[10];
 // 2ND ORDER_STAG:
 if (ORDER_STAG == 2) {
   // interpolate to point 2*i+1/2, using points 2*i-1,2*i,2*i+1
